pkg/peertls/tlsopts/tls.go

```diff
@@ -48,6 +49,12 @@ func (opts *Options) ClientTLSConfig(id storj.NodeID) *tls.Config {
 	return opts.tlsConfig(false, verifyIdentity(id))
 }
 
+// ClientTLSConfigPrefix returns a TSLConfig for use as a client in handshaking with a peer.
+// The peer node id is validated to match the given prefix
+func (opts *Options) ClientTLSConfigPrefix(idPrefix string) *tls.Config {
+	return opts.tlsConfig(false, verifyIdentityPrefix(idPrefix))
```

Comment:
Why verify the prefix instead of the whole ID?

Reply:
better UX. you can have someone type just 12vha9oTFnerx@33.20.0.1:7777 instead of something huge